ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -37,13 +37,16 @@ function wait_until_pod_ready_status() {
   start_time=$(date +%s)
   while [ $(($(date +%s) - start_time)) -lt $timeout ]; do
      pod_status="$(oc get pod -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
+     daemon_status="$(oc get daemonset -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
      echo "$pod_status"
-     if [ -n "$pod_status" ]; then
-        echo "Waiting until GPU pods of '$pod_label' in namespace '$namespace' are in running state..."
-        oc wait --timeout="${timeout_seconds}s" --for=condition=ready pod -n "$namespace" -l app="$pod_label"
+     echo "$daemon_status"
```

Comment:
In case of debugging, having also some text before the actual value and having the actual value enclosed in the `'` - this makes live easier usually:

```suggestion
     echo "Daemonset status: '$daemon_status'"
```

Reply:
See my comment: https://github.com/red-hat-data-services/ods-ci/pull/1122#discussion_r1453053922